packages/paste-core/components/typography/__tests__/heading.test.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import renderer from 'react-test-renderer';
+import {Theme} from '@twilio-paste/theme';
+import {Heading} from '../src';
+
+describe('Heading', () => {
+  it('it should render an H1 with 32px font-size', (): void => {
```

Comment:
The size totally depends on the scale implemented in the theme object and tokens.

Reply:
What do you mean?

Reply:
Just that 32px in this string doesn't map for sure to our token value based on the theme provided.  32px isn't future proof.  We should use the design language and use the token name here.